src/Assets/SVG/HeartActivedSvg.tsx

```diff
@@ -0,0 +1,30 @@
+import "./SVG.scss";
+import styled from "styled-components";
+import { onClickHeartTypeProps } from "../../Views/InteractionBar/InteractionBar";
+
+const StyledHeartActivedSvg = styled.span`
+  .HeartActivedSvg {
+    svg {
+      fill: #ff0000;
+      display: block;
+      width: 24px;
+      height: 24px;
+      padding: 12px;
+      cursor: pointer;
+    }
+  }
+`;
+
+const HeartActivedSvg = ({ onClickHeart }: onClickHeartTypeProps) => {
+  return (
+    <StyledHeartActivedSvg>
+      <span className="HeartActivedSvg" onClick={() => onClickHeart?.()}>
```

Comment:
```suggestion
const HeartActivedSvg = ({ onClickHeart: ()=>{} = ()=>{} }: onClickHeartTypeProps) => {
  return (
    <StyledHeartActivedSvg>
      <span className="HeartActivedSvg" onClick={() => onClickHeart()}>
```

이렇게 함수 기본값 미리 넣어주는 방법도 있어요